src/enums/ErrorMessages.ts

```diff
@@ -0,0 +1,8 @@
+export enum ErrorMessages {
+  noError = '',
+  fetchAll = 'Unable to fetch todos',
+  emptyValue = 'Title can\'t be empty',
+  addError = 'Unable to add a todo',
+  deleteError = 'Unable to delete a todo',
+  updateError = 'Unable to update a todo',
```

Comment:
enum's elements should started from Capital case 
```suggestion
  NoError = '',
  FetchAll = 'Unable to fetch todos',
  EmptyValue = 'Title can\'t be empty',
  AddError = 'Unable to add a todo',
  DeleteError = 'Unable to delete a todo',
  updateError = 'Unable to update a todo',
```